test/integration/test-errors.js

```diff
@@ -39,13 +42,13 @@ describe.configure().retryOnSaucelabs().run('error page', function() {
       }, () => {
         return new Error('Failed to find errors. HTML\n' +
             fixture.doc.documentElement./*TEST*/innerHTML);
-      });
+      }, timeout);
     });
   });
 
   it.configure().skipFirefox().skipEdge()
       .run('should show the body in error test', () => {
-        return expectBodyToBecomeVisible(fixture.win);
+        return expectBodyToBecomeVisible(fixture.win, timeout);
```

Comment:
You needn't in a timeout here, since 5000 is the default in `expectBodyToBecomeVisible`. I'm fine leaving it in for documentation's sake, though.